Code/Editor/Lib/Tests/test_ModularViewportCameraController.cpp

```diff
@@ -19,41 +19,43 @@ namespace UnitTest
 
     using AzToolsFramework::ViewportInteraction::MouseInteractionEvent;
 
-    class ModularViewportCameraControllerFixture : public AllocatorsTestFixture
+    class ViewportMouseCursorRequestImpl : public AzToolsFramework::ViewportInteraction::ViewportMouseCursorRequestBus::Handler
     {
     public:
-        static const AzFramework::ViewportId TestViewportId;
-
-        void SetUp() override
+        void Connect(const AzFramework::ViewportId viewportId, AzToolsFramework::QtEventToAzInputMapper* inputChannelMapper)
         {
-            AllocatorsTestFixture::SetUp();
-
-            m_rootWidget = AZStd::make_unique<QWidget>();
-            m_rootWidget->setFixedSize(WidgetSize);
-
-            m_controllerList = AZStd::make_shared<AzFramework::ViewportControllerList>();
-            m_controllerList->RegisterViewportContext(TestViewportId);
-
-            m_inputChannelMapper = AZStd::make_unique<AzToolsFramework::QtEventToAzInputMapper>(m_rootWidget.get(), TestViewportId);
+            AzToolsFramework::ViewportInteraction::ViewportMouseCursorRequestBus::Handler::BusConnect(viewportId);
+            m_inputChannelMapper = inputChannelMapper;
         }
 
-        void TearDown()
+        void Disconnect()
         {
-            m_inputChannelMapper.reset();
-
-            m_controllerList->UnregisterViewportContext(TestViewportId);
-            m_controllerList.reset();
-            m_rootWidget.reset();
-
-            AllocatorsTestFixture::TearDown();
+            AzToolsFramework::ViewportInteraction::ViewportMouseCursorRequestBus::Handler::BusDisconnect();
         }
```

Comment:
question: should ```m_inputChannelMapper```  still be reset here? its set on connect, but not sure if the reset is required.

Reply:
Ah yes it should, I think it's just moved - it is now destroyed in `TearDown` and is created in `SetUp`